tests/CMakeLists.txt

```diff
@@ -9,36 +9,43 @@ target_link_libraries(hwwrap PUBLIC chcomp)
 # Library test functions
 
 function(new_proj_test test_name compartment)
+    string(FIND ${test_name} " " test_name_delim_pos)
+    if (NOT ${test_name_delim_pos} EQUAL "-1")
+        string(SUBSTRING ${test_name} ${test_name_delim_pos} "-1" test_args)
+        string(SUBSTRING ${test_name} "0" ${test_name_delim_pos} test_name)
```

Comment:
Not sure what this second substring operation is trying to do. In any case, I think this whole `if` condition is not required.

Reply:
As described above, not required in the current incarnation (since argument passing to compartments is not yet supported), but ready implemented for when this does happen. I can definitely omit these changes, if they are undesired.

Reply:
Since there are no arguments that are being passed yet, this looks strange in this PR - but not too fussy about it. Might be an idea just to put a comment just above the if condition saying what this is doing.